### PR TITLE
chore: generated CHANGELOG for origin/release/v51.2.0

### PR DESCRIPTION
**Pull request by sfsholden**
### What does this PR do?
Changelog for v51.2.0.

### CODE REVIEW

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+# 51.2.0 - February 24, 2021
+
+## Added
+
+#### salesforcedx-vscode-apex
+
+- Add namespace support to test:run & debug commands ([PR #2961](https://github.com/forcedotcom/salesforcedx-vscode/pull/2961))
```
Comment (AnanyaJha) [inline packages/salesforcedx-vscode/CHANGELOG.md:7]: Let's add a link to the issue that was filed for this one -https://github.com/forcedotcom/salesforcedx-vscode/issues/2865
And maybe something like "Add namespace support for running and debugging tests" for the text

**Review by Dmarch28 — APPROVED** [review]

File: packages/salesforcedx-vscode/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+
+#### docs
+
+- SOQL Builder - Add LIKE support ([PR #2971](https://github.com/forcedotcom/salesforcedx-vscode/pull/2971))
+
+- SOQL Builder - Add COUNT support ([PR #2946](https://github.com/forcedotcom/salesforcedx-vscode/pull/2946))
+
+#### salesforcedx-vscode-apex
+
+- Execute anonymous apex command not focusing the output channel ([PR #2962](https://github.com/forcedotcom/salesforcedx-vscode/pull/2962)), ([Issue #2947](https://github.com/forcedotcom/salesforcedx-vscode/issues/2947))
```
Comment (ekapner) [inline packages/salesforcedx-vscode/CHANGELOG.md:19]:
```suggestion
- Execute anonymous Apex command not focusing the output channel ([PR #2962](https://github.com/forcedotcom/salesforcedx-vscode/pull/2962)), ([Issue #2947](https://github.com/forcedotcom/salesforcedx-vscode/issues/2947))
```

**Review by ekapner — COMMENTED** [review]
Just a small capitalization request

**Review by lcampos — APPROVED** [review]

**Comment by Dmarch28** [comment]
windows-Itpro-doc-master

**Pull request merged**